src/components/Person/Person.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import cx from 'classnames';
+
+import parseHtml from 'utils/parse-html';
+import { ImageElement } from 'Image';
+
+export type PersonProps = {
+  description?: string;
+  imageSizes?: string;
+  imageSrc?: string;
+  imageSrcSet?: string;
+  jobTitle: string;
+  layoutVariant: 'author' | 'full-width' | 'narrow' | 'team';
+  links?: {
+    title: string;
+    url: string;
+  }[];
+  name: string;
+  organisation?: string;
+};
+
+export const Person = ({
+  description,
+  imageSizes,
+  imageSrc,
+  imageSrcSet,
+  jobTitle,
+  layoutVariant,
+  links,
+  name,
+  organisation
+}: PersonProps) => (
+  <div
+    className={cx('cc-person', {
+      [`cc-person--${layoutVariant}`]: layoutVariant
+    })}
+    itemScope
+    itemType="http://schema.org/Person"
+  >
+    {(imageSrc || imageSrcSet) && (
+      <figure className="cc-person__image">
+        <ImageElement
+          itemProp="image"
+          sizes={imageSizes}
+          src={imageSrc}
+          srcSet={imageSrcSet}
+          alt={`A photograph of the person, ${name}.`}
+        />
+      </figure>
+    )}
+    <div className="cc-person__wrapper">
+      <div className="cc-person__header">
+        <h3 className="cc-person__heading" itemProp="name">
+          {name}
+        </h3>
+        <p
```

Comment:
I can see from the type definition that jobTitle is expected but is there any likelihood of this being an optional value? Might it need logic?

Reply:
Sure, I can add a logic to this, but at this point it is not possible to create a new Person without a job title (it is required field in the CMS).

Link to the CMS: https://wt-corporated8-develop.codeenigma.net/node/add/person